concepts/05 UI Components/Diagram/50 How To/20 Customize the Diagram Component/15 Specify a Command's Location on the Main Toolbar.md

```diff
@@ -0,0 +1,276 @@
+# Specify a Command's Location on the Main Toolbar
+
+Use the [location](/api-reference/10%20UI%20Components/dxDiagram/1%20Configuration/mainToolbar/commands/location '/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/mainToolbar/commands/location') property to set the location of a command or separator on the main toolbar. This property accepts one of the following values:
+
+- **center**  
+ Places the command in the center of the main toolbar.
+
+- **before**  
+ Places the command before the central element(s).
+
+- **after**  
+ Places the command after the central element(s).
+
+The Diagram UI component displays commands with identical location in the order in which you listed them. When the main toolbar cannot accommodate all commands, the component places excess commands in the overflow menu.
```

Comment:
What do you mean by "commands with identical location"?
Is it possible for two different commands to have the same location?

Reply:
"commands with identical location" = "commands with the same **location** property value". For instance:

`commands: [ { name: "copy", location: "center" },  { name: "paste", location: "center" }, ]`

Reply:
Can we say the following?

```suggestion
The Diagram UI component displays commands with the same value of the **location** property in the order in which you listed them. When the main toolbar cannot accommodate all commands, the component places excess commands in the overflow menu.
```

Reply:
Thanks for the clarification. How about the following?

```suggestion
The Diagram UI component displays commands with the same **location** property value in the order in which you listed them. When the main toolbar cannot accommodate all commands, the component places excess commands in the overflow menu.
```